src/EventTickets/Models/EventTicketType.php

```diff
@@ -147,6 +144,9 @@ public static function fromQueryBuilderObject($object): EventTicketType
         return new EventTicketType([
             'id' => (int)$object->id,
             'event_id' => (int)$object->event_id,
```

Comment:
@pauloiankoski just a heads up, all our models use a camelCase convention for their property names so we'll want to follow that syntax for events 😄

Reply:
@jonwaldstein I'll create another PR fixing that then I'll update this one.

Reply:
https://github.com/impress-org/givewp/pull/7233

Reply:
#7233 has been merged.